rspamd/etc/rspamd/local.d/antivirus.conf

```diff
@@ -0,0 +1,22 @@
+#
+# Antivirus -- https://rspamd.com/doc/modules/antivirus.html
+#
+
+clamav {
+    servers = "127.0.0.1:11330";
+
+    action = "reject";
+
+    scan_mime_parts = false;
+
+    # symbol to add (add it to metric if you want non-zero weight)
+    symbol = "CLAM_VIRUS";
+
+    type = "clamav";
+
+    # You can enable logging for clean messages
+    log_clean = true;
```

Comment:
do not you fear to over charge the log with the logging of clean message. I think it is not enabled in NS7

Reply:
No, I don't. The Email app writes so many log lines for each _connection_! This is just one more...

The point is that Rspamd caches ClamAV response. I think that is an important information to better understand what is happening. Did Rspamd use a cached response? Did ClamAV analyze that message? Did it reach time-out?